assets/pages/new-contributions-landing/components/ContributionForm.jsx

```diff
@@ -69,15 +68,21 @@ type PropTypes = {|
   checkoutFormHasBeenSubmitted: boolean,
   setCheckoutFormHasBeenSubmitted: () => void,
 |};
+
+type FormValueType = string | null;
+
+const getCheckoutFormValue = (formValue: FormValueType, userValue: FormValueType): FormValueType =>
+  (formValue === null ? userValue : formValue);
```

Comment:
`formValue || userValue`

Reply:
If formValue is an empty string, it will evaluate to `false`, which is exactly the issue this PR is addressing. We need it to evaluate to `true` in that instance, so that the state is update with the empty string.

Reply:
haha, yeah welcome to JavaScript